pkg/addons/data/DaemonSet.apps/openshift-template-service-broker/apiserver.yaml

```diff
@@ -1,10 +1,11 @@
 apiVersion: apps/v1
-kind: Deployment
+kind: DaemonSet
```

Comment:
I would continue using Deployments vs DaemonSets.

Reply:
We need to decide if we want to stick with Upstream version or our own version. No opinion here.